internal/arch/command-table.go

```diff
@@ -35,21 +35,23 @@ type CommandFunc func(*db.DB, []string) *protcl.Message
 type Command struct {
 	ModifyKeySpace bool
 	Fn             CommandFunc
+	MinArgs        int
+	MaxArgs        int
 }
 
 var CommandTable = map[string]Command{
 	// server
-	"ping": {ModifyKeySpace: false, Fn: cmds.Ping},
+	"ping": {ModifyKeySpace: false, Fn: cmds.Ping, MinArgs: -1, MaxArgs: 1},
```

Comment:
ping can have minimum 0 args and exactly 1 arg. So `MinArgs: 0` should be here

Reply:
done